src/app/docs/inline-delete/inline-delete-docs.component.html

```diff
@@ -0,0 +1,99 @@
+          caption="Do use inline delete when deleting an item from a list."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/inline-delete/use-list.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline
+        heading="Don't use when"
+        headingStyle="danger"
+      >
+        <p>
+          Don't use inline delete when deleting an item from outside of a list. Use a <a href="https://developer.blackbaud.com/skyux/design/guidelines/confirm">confirm</a> dialog instead.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't use inline delete when deleting an item from outside of a list."
+          captionType="danger"
+          imageSource="~/assets/img/guidelines/inline-delete/dont-use-nolist.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use a confirm dialog when deleting an item from outside of a list."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/inline-delete/dont-use-nolist-confirm.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Anatomy"
+    >
+      <sky-docs-anatomy
+        imagePath="~/assets/img/guidelines/inline-delete/anatomy.png"
+      >
+        <sky-docs-anatomy-item>Delete button</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Cancel button</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Scrim</sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Behavior"
+    >
+      <sky-docs-design-guideline
+        heading="Animation"
+      >
+        <p>
+          When the user invokes the delete action the inline delete confirmation scales up to cover the list item. When the user confirms the delete remove the item and slide the remaining items into their new positions.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          videoSource="~/assets/img/guidelines/inline-delete/delete.mp4"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+    
+    <sky-docs-demo-page-section
+      heading="Related information"
+    >
+      <sky-docs-design-guideline
+        heading="Components"
+      >
+        <ul>
+          <li><a href="https://developer.blackbaud.com/skyux/components/confirm">Confirm dialog</a></li>
+        </ul>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline
+        heading="Guidelines"
+      >
+        <ul>
+          <li><a href="https://developer.blackbaud.com/skyux/design/styles/motion">Motion</a></li>
+        </ul>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
```

Comment:
Not sure where to call this out, but the table styles on the Development tab are busted. I assume that we need to update the version of docs-tools or update how we implement the docs-tools in this repo.

Also, I don't see the JSDoc comments for the component properties in this PR, so I assume they are already in place. I think we should update the description for the `pending` property to change "Specifies if" to "Indicates whether" for consistency with how we document boolean properties throughout the docs. Also, is there any more to say about that property? It just seems like maybe there is more we could say about how that property can be used.

Finally, do we need code samples? If we decided not to include code samples, I think we should have a note explaining why. I'm not sure what our exact logic is here, but here's the best I came up with so far:

"Inline delete works with dropdowns within lists, and we do not have separate code samples for it. For examples of those components, see the [dropdown](https://developer.blackbaud.com/skyux/components/dropdown) and [list](https://developer.blackbaud.com/skyux/components/list) documentation."

Reply:
Resolving this because the table styles were resolved and because Trevor is creating code samples.